content/en/opentelemetry/setup/ddot_collector/custom_components.md

```diff
@@ -2,16 +2,13 @@
 title: Use Custom OpenTelemetry Components with Datadog Distribution of OpenTelemetry (DDOT) Collector
 aliases:
 - "/opentelemetry/agent/agent_with_custom_components"
+site_support_id: ddot_collector
```

Comment:
Is these `ddot_collector` entries in the frontmatter needed? The entry in the params file should cover it, unless there's a bug somewhere 🤔